app/controllers/test/components/mentoring/queue_controller.rb

```diff
@@ -0,0 +1,78 @@
+class Test::Components::Mentoring::QueueController < ApplicationController
+  def show
+    @sort_options = [
+      { value: "recent", label: "Sort by Most Recent" },
+      { value: "exercise", label: "Sort by Exercise" },
+      { value: "student", label: "Sort by Student" }
+    ]
+  end
+
+  def solutions
+    return head :internal_server_error if params[:state] == "Error" || params[:state] == "Loading"
+
+    results = [
```

Comment:
This will probably be converted to a serializer (see https://github.com/exercism/v3-website/pull/101) later, right?